docs/library/utime.rst

```diff
@@ -28,8 +28,8 @@ by following means:
   in such case).
 
 If actual calendar time is not maintained with a system/MicroPython RTC,
-functions below which require reference to current absolute time may
-behave not as expected.
+functions below which require reference to current absolute time may not
```

Comment:
Yes, I'd suggest to undo this specific change, then review of the rest of changes can continue (I did just a quick look thru them, and they look good, but I apologize in advance if there will be more change suggestions). I'm just trying to explain an idea behind why it was written like that.